packages/amazonq/.changes/next-release/Bug Fix-0d324321-bddd-4f93-80c4-80978930f157.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "Amazon Q /dev: include telemetry for workspace usage when generating new files"
```

Comment:
This is not something customers are going to care about. Use the `telemetry` PR title instead of `fix`. Or if there is an actual fix to mention in the changelog, please update this changelog.

Reply:
No fix, just telemetry. I thought the keywords were just limited for fix, feature, so on. I've just updated, thanks for the heads up.

Reply:
The keywords are documented here: https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#pull-request-title

And in the CI job error message.